blockchain/state/iterator.go

```diff
@@ -148,14 +153,98 @@ func (it *NodeIterator) retrieve() bool {
 	// Otherwise retrieve the current entry
 	switch {
 	case it.dataIt != nil:
-		it.Hash, it.Parent = it.dataIt.Hash(), it.dataIt.Parent()
+		it.Type = "storage"
```

Comment:
How about using `it.Type`'s own enum type?

Reply:
This `type` will be printed out in API.
So if we can use enum type, we need additional code to convert string.
I think this is enough. :)